server/routes/index.ts

```diff
@@ -0,0 +1,13 @@
+import type { RequestHandler, Router } from 'express'
+
+import asyncMiddleware from '../middleware/asyncMiddleware'
+
+export default function routes(router: Router): Router {
+  const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
```

Comment:
Some alternatives to manually writing async middleware for express:

- allow async/await functionality in express with helper fuctions:
  https://www.npmjs.com/package/@awaitjs/express
- koa is a framework by the same people as express, but designed to be more expressive and supports async/await out of the box
  https://koajs.com/

Reply:
Think maybe the lack of compatibility with other express middleware might be a bit of an obstacle in moving to koa?